pr-data.csv

```diff
@@ -3596,8 +3596,8 @@ https://github.com/sonatype-nexus-community/nexus-repository-helm,60a9e8de0d97bf
 https://github.com/soot-oss/soot,23758df0ce8327bb414d982aeb325a4f5e83ccf8,.,soot.toolkits.graph.TestDominance.TestAcyclicCFG,ID,Accepted,https://github.com/soot-oss/soot/pull/1802,
 https://github.com/soot-oss/soot,23758df0ce8327bb414d982aeb325a4f5e83ccf8,.,soot.toolkits.graph.TestDominance.TestMultiTailedPostDom,ID,Accepted,https://github.com/soot-oss/soot/pull/1802,
 https://github.com/soot-oss/soot,23758df0ce8327bb414d982aeb325a4f5e83ccf8,.,soot.toolkits.graph.TestDominance.TestSimpleDiamond,ID,Accepted,https://github.com/soot-oss/soot/pull/1802,
-https://github.com/spdx/tools,cf965bc2fa99e1c5ca23f992f18cef48c08dec9f,.,org.spdx.compare.TestPackageSheet.tesConcludedLicensetCol,ID,,,
-https://github.com/spdx/tools,cf965bc2fa99e1c5ca23f992f18cef48c08dec9f,.,org.spdx.compare.TestPackageSheet.testDeclaredLicenseCol,ID,,,
+https://github.com/spdx/tools,cf965bc2fa99e1c5ca23f992f18cef48c08dec9f,.,org.spdx.compare.TestPackageSheet.tesConcludedLicensetCol,ID,Opened,https://github.com/spdx/tools/pull/312,
```

Comment:
Isn't this merged?

Reply:
Those two tests are flaky due to the same reason, so it refer to same PR.

Reply:
Yuntong said your status should be `Accepted` (congratulations for that) not just `Opened`.

Reply:
Thanks for the reminder, I just saw it.